app/controllers/bookings_controller.rb

```diff
@@ -1,7 +1,11 @@
 class BookingsController < ApplicationController
 
   def index
-    @bookings = Booking.where(user_id: current_user.id)
+    if params[:status]
```

Comment:
I believe this will always be true cause we don't allow a booking with null status in the DB. I think `if current_user.role == "renter"` is what we need here.